app/routes/docs/about.md

```diff
@@ -5,4 +5,20 @@ meta:
 
 # About GCN
 
-Fill me in for top-level documentation page.
+The General Coordinates Network (GCN) is a public collaboration platform run by NASA for the astronomy research community to share alerts and rapid communications about high-energy, multimessenger, and transient phenomena.
+
+GCN is the established platform for publishing discoveries and follow-up of gamma-ray bursts (GRBs), gravitational-wave (GW) compact binary mergers, and high-energy neutrinos. GCN distributes alerts between space- and ground-based observatories, physics experiments, and thousands of astronomers around the world.
+
+# Documentation
+
+## Client Configuration
+
+GCN [Client Configuration](https://gcn.nasa.gov/docs/client) documentation for [**hop-client**](https://gcn.nasa.gov/docs/client#hop-client), [**adc-streaming**](https://gcn.nasa.gov/docs/client#adc-streaming) and [**confluent-kafka**](https://gcn.nasa.gov/docs/client#confluent-kafka).
+
+## Contributing
+
+GCN is an open source project built in [Remix](https://remix.run), a full stack web framework. Have an idea to make GCN better and want to contribute, or have a bug to report? Head over to our [contributing](https://gcn.nasa.gov/docs/contributing) documentation for our quick start instructions.
```

Comment:
```suggestion
GCN is an open source project built in [Remix](https://remix.run), a full stack web framework. Have an idea to make GCN better and want to contribute, or have a bug to report? Head over to our [contributing](/docs/contributing) documentation for our quick start instructions.
```

Reply:
Also clarify that the GCN _web site_ is built in Remix. Although our contributing guide currently only talks about how to get involved with contributing to the web site.